test/unit/mocks/logger.js

```diff
@@ -0,0 +1,10 @@
+'use strict'
+const sinon = require('sinon')
+
+module.exports = (sandbox = sinon) => ({
```

Comment:
figured we could re-use this mock, updated the one place I was using it before

Reply:
Nice add. Super helpful!